profile.d/toolbox.sh

```diff
@@ -6,6 +6,7 @@ host_welcome_stub="$toolbox_config/host-welcome-shown"
 toolbox_welcome_stub="$toolbox_config/toolbox-welcome-shown"
 
 if [ -f /run/ostree-booted ] \
+   && [ -d /run/gdm ] \
```

Comment:
Since AFAIK Endless is also packaging this project, I'd instead do something like:

```
. /usr/lib/os-release
...
 && [ "${ID}" = "fedora" -a "${VARIANT_ID}" = "workstation" -o "${VARIANT_ID}" = "silverblue" ]
```

or so.

Also I'd just repeat I'm still developing https://github.com/cgwalters/coretoolbox since I don't think bash is appropriate for this.

Reply:
And probably a better fix is to move this bit into https://pagure.io/workstation-ostree-config/ anyways.

Reply:
> And probably a better fix is to move this bit into https://pagure.io/workstation-ostree-config/ anyways.

@cgwalters Move which bits? the bash profile?

Reply:
Right.

Reply:
In workstation-ostree-config, `toolbox` is not included in the f25-f29 branches.  It doesn't make sense to move the profile unless we add toolbox to the configs in those branches also.  For now I just update this PR to check os-release.